pages/common/mysqldump.md

```diff
@@ -5,11 +5,11 @@
 
 - Create a backup, user will be prompted for a password:
 
-`mysqldump -u {{user}} --password {{database_name}} -r {{filename.sql}}`
+`mysqldump -u {{user}} -p {{database_name}} -r {{filename.sql}}`
 
 - Restore a backup, user will be prompted for a password:
 
-`mysql -u {{user}} --password -e "source {{filename.sql}}" {{database_name}}`
+`mysql -u {{user}} -p {{database_name}} < {{filename.sql}}`
```

Comment:
I'd probably argue there that it's ambiguous as to whether the `{{database_name}}` applies to the `-p` or not if we shorten it. I'd suggest that we stick with the longer one in this instance.

Reply:
Agree with @sbrl here. Also, I just noticed that the command is `mysql` and not `mysqldump`, that looks like an error, care to fix that too please?

Reply:
I don't have a preference between `-p` and `--password`... but command in this example is `mysql` and not `mysqldump`, that looks like an error, care to fix that too please?

Reply:
Ups! @mebeim is right! @sbrl, I think it might be solved editing your own suggestion.

Reply:
I can certainly generate a suggestion. My original comment doesn't have one though.
```suggestion
`mysqldump --user {{user}} --password {{database_name}} < {{path/to/file.sql}}`
```